dist/build.gradle

```diff
@@ -71,12 +76,28 @@ task copyLib(group: 'Build',
     }
 }
 
+task downloadClientBinaries(group: 'Build',
+        description: "Downloads client binaries into ${project.ext.cliDownloadDir}") {
+
+    outputs.dir("${project.ext.cliDownloadDir}")
+
+    doLast {
+        ["darwin-amd64", "linux-amd64", "windows-amd64.exe"].each { platform ->
+            download.run {
+                src "https://github.com/line/centraldogma-go/releases/download/0.0.1/dogma.$platform"
+                dest "${project.ext.cliDownloadDir}/dogma.$platform"
+            }
+        }
+    }
+}
+
 task copyClientBinaries(group: 'Build',
-        description: "Copies client binaries into ${project.ext.relativeDistDir}/lib",
-        dependsOn: project(':cli').tasks.assemble,
-        type: Copy) {
-    from project(':cli').ext.clientBinDir
+        description: "Copies client binaries into ${project.ext.relativeDistDir}/lib", type: Copy) {
```

Comment:
This is not your fault but let's change to `${project.ext.relativeDistDir}/bin/native`

Reply:
I was also wondering what the folder meant. 😆